src/test/java/com/commercetools/emailprocessor/utils/ConfigurationUtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package com.commercetools.emailprocessor.utils;
+
+import com.commercetools.emailprocessor.model.ProjectConfiguration;
+import org.junit.Test;
+
+import java.util.Optional;
+
+import static com.commercetools.emailprocessor.utils.ConfigurationUtils.getConfiguration;
+import static org.junit.Assert.assertEquals;
+
+public class ConfigurationUtilsTest {
```

Comment:
You can also test for:
1. relative resource path
2. not valid json config file

Reply:
line 29